pkg/apis/navigator/validation/cassandra.go

```diff
@@ -40,6 +40,11 @@ func ValidateCassandraClusterUpdate(old, new *navigator.CassandraCluster) field.
 				if !reflect.DeepEqual(newNp.NodeSelector, oldNp.NodeSelector) {
 					allErrs = append(allErrs, field.Forbidden(idxPath.Child("nodeSelector"), "cannot modify nodeSelector"))
 				}
+
+				if !reflect.DeepEqual(newNp.Persistence, oldNp.Persistence) {
```

Comment:
If persistence was previously disabled, we probably can allow changes that enable the field later?

Reply:
I've allowed users to enable, but not disable